Lesson about sprint board best practices (#799)
Ada-Developers-Academy/textbook-curriculum: 00-programming-fundamentals/using-planning-boards.md

    - For group projects, this means focusing on one story at a time per person.
1. Use checklists on each user story to break the story up into smaller tasks. 
    - This makes it easier to ensure every task you’re working on is directly related to the user story you’re currently focused on.
    - This way, with each user story you complete, you know you already have this much to show off for your demo!
1. Organize Trello cards by “Iteration/Week”. 
    - This way you have some idea how you’re tracking: whether you’re ahead, behind, or on-track. 
    - The easiest way to do this is probably to estimate what percentage of your project each user story represents and set a due date for it accordingly. 
      - eg. If a story represents ~30% of what you have left to do for the project, it should be due in 13 days * .30 = 3.9 days (or around to 4 days).
1. If there are one or two user stories that feel the scariest or riskiest, see if you can do those first. 
    - That will give you more time to make mistakes and it will give you more time to change your idea if something seems like it isn’t quite going to work like you expected.

>   ## Exercise

Reviewer: Maybe we give students an example Trello to examine and reference for these best-practices.

Reviewer 2: That, or pictures! Screenshots are cheap!

Author: The example board exists just a couple lines below.

# Using Planning Boards

## Learning Goals
- Understand Ada's recommended best-practices for using planning boards in Trello
- Practice creating a planning board with tasks scheduled by iteration

## Prerequisite
This lesson will be hard to follow if students haven't first received the [Intro to Agile lesson](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/intro-to-agile.md)

> ## Ada Best Practices

Reviewer 2: Maybe we should call this something else? Maybe...
```suggestion
## Ada Suggested Practices
```

1. Create cards as User Stories 
    - Format: "As a user, I can ...[, so that I...]". 
        - eg. "As a user, I can browse products by merchant, so that I can explore all the cool stuff my favorite merchant sells and buy more stuff."

>     - This helps you as a developer focus on what matters to the user. Without articulating your work this way, it's SO EASY to get distracted by lots of technical issues that may or may not actually matter to the user your application is meant to serve.

Reviewer 2: ```suggestion
    - This helps you as a developer focus on what matters to the user. Without articulating your work this way, it's SO EASY to get distracted by lots of technical issues that may or may not actually matter to the end user!
```

1. Sort user stories by priority and approach the project by focusing on completing one story at a time.
    - For group projects, this means focusing on one story at a time per person.
1. Use checklists on each user story to break the story up into smaller tasks. 
    - This makes it easier to ensure every task you’re working on is directly related to the user story you’re currently focused on.

>     - This way, with each user story you complete, you know you already have this much to show off for your demo!

Reviewer 2: ```suggestion
    - This way, with each user story you complete, you know you already have this much to show off for your demo!
    - If something in the checklist ends up being a big task in its own right, you can turn individual checklist items into their own cards!
```

1. Organize Trello cards by “Iteration", week, or priority. 
    - Find some way of setting due dates for a piece of work

>       - eg. If a story represents ~30% of what you have left to do for the project and you have 13 more working days, it should be due in 13 days * .30 = 3.9 days (or around to 4 days).

Reviewer 2: "Around to 4 days"
```suggestion
      - eg. If a story represents ~30% of what you have left to do for the project and you have 13 more working days, it should be due in 13 days * .30 = 3.9 days (or around 3 to 4 days).
```
